papaslib/CMakeLists.txt

```diff
@@ -1,24 +1,11 @@
 cmake_minimum_required(VERSION 2.8)
 
 # use, i.e. don't skip the full RPATH for the build tree
-SET(CMAKE_SKIP_BUILD_RPATH  FALSE)
+SET(CMAKE_SKIP_BUILD_RPATH  TRUE)
```

Comment:
Just to answer why I was asking Alice to disable setting the RPATH: It not only hardcodes the dependent library paths into the library but also the location of the `papaslib` to be `CMAKE_INSTALL_PREFIX`. This is normally not a problem, but gets us into trouble for the cvmfs installations where we cannot build+install easily with the prefix set to /cvmfs/... So it is important for us to have the libraries re-locatable. Setting the RPATH breaks that to some extend (without additional post-install cleaning).